tests/reuse_file_descriptor.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2020 Alibaba Cloud. All rights reserved.
```

Comment:
nit: Naming the test module `reuse_file_descriptor` would mean that this file will always have just one test. Maybe we can rename it to `regressions` or something else so that we can add more tests to it later.